index.js

```diff
@@ -89,15 +98,15 @@ function CogLvlCell(cogLvl) {
 function CogLvlColumn() {
   let arr = [];
   // TODO For lvl 13+ cogs
-  for (let cogLvl = 12; cogLvl >= 1; cogLvl--) {
+  for (let cogLvl = 20; cogLvl >= 1; cogLvl--) {
     arr.push(CogLvlCell(cogLvl));
   }
   return arr.join('');
 }
 
 function CombosGrid() {
   const arr = [];
-  for (let cogLvl = 12; cogLvl >= 1; cogLvl--) {
+  for (let cogLvl = 20; cogLvl >= 1; cogLvl--) {
```

Comment:
Same thing here. Maybe since it's used in multiple places, we specify this as a function somewhere:
```js
const getMaxCogLvl = () => state.game === 'ttr' ? 20 : 12;
```

Reply:
good idea, implemented.